packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -734,6 +736,7 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
             this.setSubmitting(false);
 
             if (response?.rows) {
+                await incrementClientSideMetricCount(ENTITY_CREATION_METRIC, nounPlural + 'CreationFromGrid');
```

Comment:
This is my first time seeing `incrementClientSideMetricCount()`. It's neat. From an error handling standpoint we may want to consider having that method default to failing silently so we don't actually prevent applications from working in the event that it does fail for some reason.

Additionally, from a testing perspective, I think it would be nice to have this call be a part of the component APIs so we can stub out the requests.

Reply:
Both good suggestions. Done.